src/components v2/Atoms/Components/SkipLink/index.ts

```diff
@@ -0,0 +1,3 @@
+import P2SkipLink from './SkipLink.vue';
+
+export default P2SkipLink;
```

Comment:
we should export like this

export {default, props} from './SkipLink.vue';

Reply:
Resolved